components/sysutils/glances/Makefile

```diff
@@ -0,0 +1,80 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright 2022, Benny Lyons
+#
+
+BUILD_BITS=NO_ARCH
+BUILD_STYLE=setup.py
```

Comment:
Hmmmm.  This seems to be a nice candidate for `python-integrate-project` integration.  Did you try to integrate using `$WS_TOOLS/python-integrate-project Glances`?  It might be a bit harder to create the first integration, but all future maintenance and upgrades would be far far easier.